rmc/utils/generic.py

```diff
@@ -254,12 +255,9 @@ def filter_context_using_gnomad(
     :param gnomad_data_type: gnomAD data type. Used to retrieve public release and coverage resources.
         Must be one of "exomes" or "genomes" (check is done within `public_release`).
         Default is "exomes".
-    :param adj_freq_index: Index of frequency array that contains global population filtered calculated on
+    :param adj_freq_index: Index of frequency array that contains global population calculated on
         high quality (adj) genotypes. Default is 0.
```

Comment:
Phrasing that may be clearer: `Index of array containing allele frequencies calculated on high-quality (adj) genotypes over various population groupings. Default is 0.`

Reply:
ah yes, I'm used to thinking about this as global from QC. will update!